linux-requirements.txt

```diff
@@ -1 +1,8 @@
 pygobject~=3.22.0
+
+To install, run the following commands on terminal:
```

Comment:
These instructions are for linux systems only. So shall I create a separate section for Linux in readme? Another way would be mention the requirements files in readme so that readme remains concise and relevant. Also when instructions for other OSes are found, their instructions-files can be updated.